internal/defaults.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Defaults for rules_typescript repository not meant to be used downstream"
+
+load("@build_bazel_rules_typescript//:defs.bzl", _ts_library = "ts_library")
+
+DEFAULT_TS_LIBRARY_COMPILER = "@build_bazel_rules_typescript//internal:tsc_wrapped_bin"
```

Comment:
comment explaining that we can't use the default because that would be a JS dependency on npm, and we are building the thing itself to publish it to npm

Reply:
done